app/src/main/java/org/mifos/mobile/ui/activities/RegistrationActivity.kt

```diff
@@ -24,11 +24,11 @@ class RegistrationActivity : BaseActivity() {
             .setTitle(getString(R.string.dialog_cancel_registration_title))
             .setMessage(getString(R.string.dialog_cancel_registration_message))
             .setPositiveButton(
-                getString(R.string.cancel),
+                getString(R.string.yes),
                 DialogInterface.OnClickListener { _, _ -> super.onBackPressed() },
```

Comment:
you can remove the Redundant SAM-constructor here

Reply:
Okay